src/Ormolu/Parser/CommentStream.hs

```diff
@@ -46,21 +48,19 @@ import Ormolu.Utils (onTheSameLine, showOutputable, unSrcSpan)
 newtype CommentStream = CommentStream [RealLocated Comment]
   deriving (Eq, Data, Semigroup, Monoid)
 
--- | Create 'CommentStream' from 'GHC.PState'. The pragmas are
+-- | Create 'CommentStream' from 'PState'. The pragmas are
```

Comment:
It looks like there is a discrepancy between the haddock and the type signature of the function. It doesn't take `PState` anymore, only `HsModule`. This means the per-argument comment also should be updated?